tests/functional/control.py

```diff
@@ -1,3 +1,25 @@
+#!/usr/bin/env python
+#############################################################################
```

Comment:
@bazsi And let me also add a nitpick of my own. I would generally recommend setting the executable bit on all of the scripts which could be meaningfully executed from the command line.